### ISSUE TITLE
Windows 11 ARM64 Drive fails to boot. Win10 succeeds. 

### ISSUE THREAD

**Issue by protocolture** [pbatard/rufus#2208]
﻿

Checklist
---------
- [x] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [x] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [x] I clicked the 'Log' button or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, or used [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview), and copy/pasted the log into the section that says `<FULL LOG>` below.
- [x] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [ ] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [ ] I also tried one or more of the following:
  - [x] Using a different USB drive.
  - [x] Plugging the USB into a different port.
  - [ ] Running Rufus on a different computer.
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
Lenovo X13s Arm 64 Laptop. 

1. Can boot a Rufus'd Windows 10 Pro ARM64 flash drive.
2. Cannot boot a Rufus'd Windows 11 Pro ARM64 flash drive. 

Have tested multiple drives. Old San Disks, Old Data Travellers, and a brand new San Disk to top it off (this is whats featured in the logs), Secure Boot on and off, reaching out to lenovo support etc. 

Where the boot fails, and perhaps this is more related to your efi boot project, feel free to redirect me  is "Launching 'efi\boot\bootaa64.efi'

The system then reboots. 

Log
Rufus Version: 3.21.1949
---
```
Hello I hit the 65536 character limit so I have attached the log files. Apologies if this is not ideal. I did try submitting anyway but Github would not allow it. 

```

[rufus-win10.txt](https://github.com/pbatard/rufus/files/11047988/rufus-win10.txt)
[rufus-win11.txt](https://github.com/pbatard/rufus/files/11047991/rufus-win11.txt)

**Comment by pbatard** [comment]
There's absolutely **no** difference between how Rufus launches the Windows 10 ARM64 bootloader and the Windows 11 ARM64 bootloader, so if one works and the other doesn't, it's entirely between Lenovo and Microsoft.

Especially, it you see `Launching 'efi\boot\bootaa64.efi` it means that the UEFI:NTFS chain loader of Rufus has successfully launched the Microsoft bootloader, and what happens then is entirely controlled by the Microsoft code.

As you may expect, especially on ARM64 machines that are fairly recent, Microsoft has to perform basic hardware initialisation on very heterogeneous CPUs that they may or may not have properly tested and that may or may not provide extensions that Microsoft expects to find (for instance, the latest feature upgrade of Windows 11 dropped support for ARM64 v8.0 CPUs and now requires a ARM64 v8.1 or later, [which broke support for a few _not-so-old_ ARM64 systems like the Raspberry Pi 4](https://twitter.com/never_released/status/1576852301674778624)).

In short, you cannot expect something that worked with Windows 10 to also work with Windows 11, and if you do find an issue when it comes to running Windows 11 on Lenovo hardware, then you need to get in touch with either Microsoft or Lenovo, as I can vouch that this is not something that Rufus can do anything about since we're simply executing the Microsoft provided bootloaders and our chain loaders are UEFI compliant (which means that, if the issue is with chain loading, then the problem is still Lenovo or Microsoft to fix).

As such I will close this issue.
